### PR TITLE
DD-422 posting same license error

### PR DESCRIPTION
**Pull request by JingMa87**
**What this PR does / why we need it**: It fixes the 500 error when you post the same license twice.

**Which issue(s) this PR closes**: DD-422

Closes #

**Special notes for your reviewer**: N/A

**Suggestions on how to test this**: Post the same license twice.

**Does this PR introduce a user interface change? If mockups are available, please link/include them here**: No

**Is there a release notes update needed for this change?**: No

**Additional documentation**: N/A

### CODE REVIEW

File: src/main/java/edu/harvard/iq/dataverse/api/Admin.java [hunk 1]
```diff
@@ -391,40 +381,9 @@ private Response deleteAuthenticatedUser(AuthenticatedUser au) {
         return ok("AuthenticatedUser " + au.getIdentifier() + " deleted. ");
 
     }  
+    
 
-    @POST
```
Comment (janvanmansum) [inline src/main/java/edu/harvard/iq/dataverse/api/Admin.java]: Why is this section of code deleted? Doesn't seem to have anything to do with licenses.

File: src/main/java/edu/harvard/iq/dataverse/api/Admin.java [hunk 2]
```diff
@@ -936,27 +891,13 @@ public Response listBuiltinRoles() {
 		}
 	}
 
-    @DELETE
```
Comment (janvanmansum) [inline src/main/java/edu/harvard/iq/dataverse/api/Admin.java]: Idem.

**Review by janvanmansum — COMMENTED** [review]
@JingMa87 : it seems you are removing some parts of the code that have nothing to do with licenses. Please, have another look.

**Comment by JingMa87** [comment]
@janvanmansum These are the changes that got merged from the multi-license branch. I'll have to manually adjust the code because git is terrible. Also, how come there was a merge conflict in the first place?

**Comment by JingMa87** [comment]
@janvanmansum I merged the branches. Also, how come there was a merge conflict in the first place?

**Pull request merged**